example_app_generator/generate_app.rb

```diff
@@ -26,20 +26,29 @@
   gsub_file "Gemfile", /.*gem..sqlite3.*/, "gem 'sqlite3', '~> 1.3.6'"
   gsub_file "Gemfile", /.*bootsnap.*/, ""
 
-  if Rails::VERSION::STRING >= '5.0.0'
```

Comment:
Leave this, we're still soft supporting 4.2 and this makes it tricky to properly install locally.